UnitsNet.Tests/CustomCode/SpecificEnergyTests.cs

```diff
@@ -27,9 +27,9 @@ public class SpecificEnergyTests : SpecificEnergyTestsBase
     {
         protected override double JoulesPerKilogramInOneJoulePerKilogram => 1.0;
 
-        protected override double CaloriesPerGramInOneJoulePerKilogram => 1.0/4.184E3;
+        protected override double CaloriesPerGramInOneJoulePerKilogram => 2.410800386e-4;
```

Comment:
Computing this value gives me: `0.00023900573`
Was the function not correct to begin with or is this `IT` vs `th` units?
In retrospect, we maybe should have added `It` and `Th` suffixes to the unit names to disambiguate them.

Reply:
Looks like he divided with 41**48** instead of 41**84**. I'm fixing this now.

And perhaps we should either say in README that we don't support IT's but only TH's, or we should name them as you said.